ramlfications/loader.py

```diff
@@ -50,9 +50,14 @@ def _parse_json(self, jsonfile, base_path):
             base_path = base_path + "/"
         base_path = "file://" + base_path
 
-        with open(jsonfile, "r") as f:
-            schema = jsonref.load(f, base_uri=base_path, jsonschema=True)
-        return schema
+        try:
+            with open(jsonfile, "r") as f:
+                schema = jsonref.load(f, base_uri=base_path, jsonschema=True)
+        except ValueError as err:
+            msg = "ERROR: failed to parse JSON file: {0} ({1})"
+            raise LoadRAMLError(msg.format(jsonfile, err))
```

Comment:
The format is being done in this line ;) I did do it this way so the msg line does not have to be split in two lines to obey the PEP8 80 char limit, If you want me to change this, please let me know. Will look for adding a test but that will probably somewhere next week, pretty tight schedule for this week..